practice-app/app/user.py

```diff
@@ -88,3 +89,8 @@
     "x-rapidapi-host" :"google-search3.p.rapidapi.com"
 }
 
+@user_api.route('/api/v1.0/users/<id:user_id>/followers', methods=['GET'])
+def get_followers(user_id):
+    followers = session.query(Following).filter(Following.followingID == user_id).all()
+    return jsonify({"followerIDs": list(followers)}), 200
```

Comment:
I think this should be something like that because Following object is not automatically serializable.
```suggestion
    return jsonify({"followerIDs": [follower.followerID for follower in followers]}), 200
```